inc/class-saml.php

```diff
@@ -706,6 +705,25 @@ public function createUser( $username, $email ) {
 		return [ $user_id, $username ];
 	}
 
+	/**
+	 * @param string $net_id
+	 * @param array $attributes
+	 *
+	 * @return string
+	 */
+	public function getEmail( $net_id, $attributes ) {
+		if ( isset( $attributes[ self::SAML_MAP_FIELDS['mail'] ][0] ) ) {
```

Comment:
Should this be [ self::SAML_MAP_FIELDS['mail'] ][1] instead? I'm not sure how array mapping is supposed to work here.